src/utils/maps/zoomToProjectSite.ts

```diff
@@ -22,9 +23,8 @@ export default function zoomToProjectSite(
     const isMobile = window.innerWidth <= 767 && isPortrait;
     //console.log("zoomToProjectSite", viewport, viewport.width, window.innerWidth, viewport.height, window.innerHeight);
 
-    const bbox = turf.bbox(geoJson.features[selectedSite]);
+    const bbox = turf.bbox(geoJson && geoJson.features[selectedSite]);
```

Comment:
I'd suggest defining a more specific type for `geoJson`. Use the API responses and the types defined by `turf` as a guide.